schemas/context/webpage.schema.json

```diff
@@ -50,17 +50,17 @@
           "type": "object",
           "description": "The referrer of the web page.",
           "properties": {
-            "URL": {
+            "xdm:URL": {
               "title": "URL",
               "type": "string",
               "description": "The referrer URL."
             },
-            "domain": {
+            "xdm:domain": {
```

Comment:
Why do you need to separate out the domain if you have the full URL?

Reply:
@jwen-adobe, can you answer this for @lrosenthol?  I am also unclear why this is needed.

Reply:
This field was added from Analytics. I do not know Chris Luby's public repo id, will ask him through email to see if we can remove this. @kstreeter

Reply:
@kstreeter @lrosenthol xdm:domain is removed for now after communicating with analytics.